### PR TITLE
Update Portuguese Translation

### PR DESCRIPTION
**Pull request by altiereslima**

### CODE REVIEW

File: files/lang/pt.po [hunk 1]
```diff
@@ -7806,23 +7809,23 @@ msgstr ""
...
-#, fuzzy
 msgid ""
 "The Sphinx asks you the following riddle:\n"
 "\n"
 "'"
 msgstr ""
-"A Esfinge lhe pergunta o seguinte enigma:\n"
-"\n"
-"'%{riddle}'\n"
+"A Esfinge lhe pergunta o seguinte enigma::\n"
```
Comment (ihhub) [inline files/lang/pt.po:7817]: Is **::** a valid usage in Italian?

**Review by ihhub — COMMENTED** [review]
Hi @altiereslima , I left one question here. Can you please take a look?

File: files/lang/pt.po (hunk 1 above)
Comment (altiereslima) [reply files/lang/pt.po]: single colon : is the correct form

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@altiereslima , thank you so much for this update!

**Pull request merged**